src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo:React.FC<Props> = ({ todo }) => {
+  return (
+    <div
+      data-cy="Todo"
+      className={classNames('todo', { completed: todo.completed })}
+    >
+      <input
+        id="label"
+        data-cy="TodoStatus"
+        type="checkbox"
+        className="todo__status"
+        defaultChecked={todo.completed}
+      />
```

Comment:
Should have a `<label>` with `className` `todo__status-label` wrapping this `<input>`. Without it the checkbox is invisible.

Yours:
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/52002856/b5a72cc2-838b-4a72-9aa2-6b94602f5932)


Expected:
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/52002856/8e3e28a1-e811-4776-93aa-190849577081)